CRM/ManageLetterheads/Page/LetterheadsListPage.php

```diff
@@ -20,25 +20,48 @@ class CRM_ManageLetterheads_Page_LetterheadsListPage extends CRM_Core_Page {
    */
   public function run() {
     $this->availableForLabels = LetterheadAvailability::buildOptions('available_for');
+    $pager = $this->getPager();
+    list($offset, $limit) = $pager->getOffsetAndRowCount();
 
-    $this->assign('letterheads', $this->getLetterheads());
+    $this->assign_by_ref('pager', $pager);
+    $this->assign('letterheads', $this->getLetterheads($offset, $limit));
 
     parent::run();
   }
 
+  /**
+   * Returns the pager object that can be used to paginate letterheads.
+   *
+   * @return CRM_Utils_Pager
+   */
+  private function getPager() {
+    $totalLetterheads = civicrm_api3('Letterhead', 'getcount', []);
```

Comment:
no need to pass an empty array as the third parameter. It's an empty array by default